examples/splunk-hec-traces/tracing/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-stretch
+FROM golang:1.17.7-stretch
```

Comment:
I think we can leave this as `1.17` since it gets re-tagged with the latest patch version.